packages/ocs/dashboards/block-pool/states.ts

```diff
@@ -26,36 +20,29 @@ export const ImageStateLegendMap = (
   [ImageStates.UNKNOWN]: t('Unknown'),
 });
 
-enum ImageHealth {
-  OK = 'OK',
-  WARNING = 'WARNING',
-  ERROR = 'ERROR',
-  UNKNOWN = 'UNKNOWN',
-}
-
-export const healthStateMapping: { [key in ImageHealth]: HealthMappingValues } =
-  {
-    [ImageHealth.OK]: {
-      health: ImageHealth.OK,
-      icon: <GreenCheckCircleIcon />,
-    },
-    [ImageHealth.UNKNOWN]: {
-      health: ImageHealth.UNKNOWN,
-      icon: <GrayUnknownIcon />,
-    },
-    [ImageHealth.WARNING]: {
-      health: ImageHealth.WARNING,
-      icon: <YellowExclamationTriangleIcon />,
-    },
-    [ImageHealth.ERROR]: {
-      health: ImageHealth.ERROR,
-      icon: <RedExclamationCircleIcon />,
-    },
-  };
+export const MirroringImageHealthMap = {
+  0: HealthState.OK,
+  1: HealthState.UNKNOWN,
+  2: HealthState.WARNING,
+  3: HealthState.ERROR,
+};
 
-type HealthMappingValues = {
-  icon: React.ReactNode;
-  health: ImageHealth;
+export const healthStateMessage = (
+  state: keyof typeof HealthState,
+  t: TFunction
+): string => {
+  switch (state) {
+    case HealthState.OK:
+      return t('Ok');
+    case HealthState.UNKNOWN:
+      return t('Unknown');
+    case HealthState.WARNING:
+      return t('Warning');
+    case HealthState.ERROR:
+      return t('Error');
+    default:
+      return t('-');
```

Comment:
no need to translate this...
```suggestion
      return '-';
```